tests/vm_monitoring_test.go

```diff
@@ -52,40 +53,45 @@ var _ = Describe("Health Monitoring", func() {
 		tests.BeforeTestCleanup()
 	})
 
-	Context("Watchdog device", func() {
-		It("should cause VM to shutdown when watchdog expires", func(done Done) {
-			vm := tests.NewRandomVMWithWatchdog()
-			Expect(err).ToNot(HaveOccurred())
-			launchVM(vm)
-
-			expecter, _, err := tests.NewConsoleExpecter(virtClient, vm, "serial0", 10*time.Second)
-			Expect(err).ToNot(HaveOccurred())
-			defer expecter.Close()
-
-			_, err = expecter.ExpectBatch([]expect.Batcher{
-				&expect.BExp{R: "Welcome to Alpine"},
-				&expect.BSnd{S: "\n"},
-				&expect.BExp{R: "login"},
-				&expect.BSnd{S: "root\n"},
-				&expect.BExp{R: "#"},
-				&expect.BSnd{S: "watchdog -t 2000ms -T 4000ms /dev/watchdog && sleep 5 && killall -9 watchdog\n"},
-				&expect.BExp{R: "#"},
-				&expect.BSnd{S: "echo $?\n"},
-				&expect.BExp{R: "0"},
-			}, 250*time.Second)
-			Expect(err).ToNot(HaveOccurred())
-
-			namespace := vm.ObjectMeta.Namespace
-			name := vm.ObjectMeta.Name
-
-			Eventually(func() v1.VMPhase {
-				startedVM, err := virtClient.VM(namespace).Get(name, metav1.GetOptions{})
+	Describe("Watchdog device", func() {
+		Context("when expires", func() {
```

Comment:
You can remove the context and just write e.g.

It("should be shut down when the watchdog expires")